RecoTracker/LSTCore/src/alpaka/Quintuplet.h

```diff
@@ -1699,8 +1690,27 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
                                   const float ptCut) const {
       ALPAKA_ASSERT_ACC((alpaka::getWorkDiv<alpaka::Grid, alpaka::Blocks>(acc)[1] == 1) &&
                         (alpaka::getWorkDiv<alpaka::Grid, alpaka::Blocks>(acc)[2] == 1));
+
+      int& matchCount = alpaka::declareSharedVar<int, __COUNTER__>(acc);
+
+      const auto threadIdx = alpaka::getIdx<alpaka::Block, alpaka::Threads>(acc);
+      const auto blockDim = alpaka::getWorkDiv<alpaka::Block, alpaka::Threads>(acc);
+
+      const int threadIdX = threadIdx.x();
+      const int threadIdY = threadIdx.y();
+      const int blockSizeX = blockDim[2];
+      const int blockSizeY = blockDim[1];
```

Comment:
The current approach is technically correct, but maybe it would be more clear to use consistently the notation `[2]` or the notation `.x()` ?

 ```suggestion
      const int threadIdX = threadIdx.x();
      const int threadIdY = threadIdx.y();
      const int blockSizeX = blockDim.x();
      const int blockSizeY = blockDim.y();
```